### PR TITLE
Consolidation of: Create/Cancel/Updgrade account, subusers, 2FA, reset username/password

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
- [x] Taken all valuable information from existing pages and added it to the new category
- [x] Deleted old pages
- [x] Updated the glossary
- [x] Added redirects from deleted page to new content
- [x] Head data should have SEO title, keywords, and description
~- [ ] Screenshots updated~
~- [ ] all images have an alt tag~
~- [ ] all images are named according to style guide~
- [x] Content is in Jobs-to-be-done format
- [x] Update all in-content links and links to deleted pages
- [x] Added additional resources (aim for a link to the glossary, the blog and other jobs-to-be-done)
- [x] Added a TOC
- [ ] Added a talk to Dewey/Olivia section
- [ ] title optimized for google search snippits
- [ ] content optimized for google search snippits
- [ ] keyword optimization for each category
- [ ] addressed/closed any related issues in the backlog
- [ ] pages are have the correct weight

### CODE REVIEW

File: source/User_Guide/Account_Management/create_account.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/create_account.md]: BOOM

Comment (ksigler7) [inline source/User_Guide/Account_Management/create_account.md]: boom

File: source/User_Guide/Account_Management/reset_username_password.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+weight: 50
+title: Resetting your account credentials
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/reset_username_password.md]: I would call this Resetting your Username and Password

File: source/User_Guide/Account_Management/reset_username_password.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+navigation:
+  show: true
+seo:
+  title: Resetting your account credentials
+  override: true
+  description: Steps for resetting your username and/or password using the SendGrid UI
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/reset_username_password.md]: ~and/or~ and

File: source/User_Guide/Account_Management/reset_username_password.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+- [Changing your password](#-Changing-your-password)
+    - [Changing your password through the SendGrid UI](#-Changing-your-password-through-the-SendGrid-UI)
+    - [Changing your password via Forgot Password](#-Changing-your-password-via-Forgot-Password)
+- [Changing your username](#-Changing-your-username)
+
+{% anchor h2 %}
+Changing your password
+{% endanchor %}
+
+There are two ways to change your password. Through the SendGrid UI and through the Forgot Password page. For more information on password requirements and security see the [password documentation](https://sendgrid.com/docs/Classroom/Basics/Security/password.html).
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/reset_username_password.md]: ~There are two ways to change your password. Through the SendGrid UI and through the Forgot Password page.~

If you already know your password, you can change your SendGrid password by editing your [Account Details](https://app.sendgrid.com/settings/account) in the UI. If you've forgotten your password, you can reset it on the [Reset password page](https://app.sendgrid.com/forgot_password).

File: source/User_Guide/Account_Management/reset_username_password.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+When you change your account password, your SMTP and API access will change as well. Make sure you update your account information on all systems and software.
+{% endwarning %}
+
+{% anchor h3 %}
+Changing your password through the SendGrid UI
+{% endanchor %}
+
+*To change your password through the SendGrid UI:*
+
+1. From the homepage, select the drop-down in the top left corner and then select **Account Details** or navigte to **Settings** and then click **Account Details**.
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/reset_username_password.md]: *navigate

File: source/User_Guide/Account_Management/reset_username_password.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+{% endanchor %}
+
+*To change your password through the SendGrid UI:*
+
+1. From the homepage, select the drop-down in the top left corner and then select **Account Details** or navigte to **Settings** and then click **Account Details**. 
+1. Locate the *Username* and *Password* section and click **Edit**. 
+1. Enter your current password and a new password in the fields and then click **Save**.
+
+{% anchor h3 %}
+Changing your password via Forgot Password
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/reset_username_password.md]: Resetting your password

File: source/User_Guide/Account_Management/reset_username_password.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+Changing your username
+{% endanchor %}
+
+1. From the homepage, select the drop-down in the top left corner and then select **Account Details** or navigate to **Settings** and then click   **Account Details**. 
+1. Click **Edit**. 
+1. Enter a new username and then click **Save**.
+
+{% warning %}
+When you change your account username, your SMTP and API access will change as well. Make sure you update your account information on all systems and software.
+{% endwarning %}
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/reset_username_password.md]: I would add a section for resetting your username: https://support.sendgrid.com/hc/en-us/requests/new#forgot-username

File: source/User_Guide/Account_Management/reset_username_password.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+{% endanchor %}
+
+If for some reason you cannot remember your password and cannot log in, you can still change your password. 
+
+*To reset your password using the Forgot Password link:*
+
+1. Navigate to the [SendGrid login page]({{site.site_url}}/login). 
+1. click **[Forgot Password]({{site.site_url}}/user/forgotPassword)**.
+
+From here, you will need to enter your SendGrid username. We will send you an email with account reactivation steps, please follow these steps to access your account again.
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/reset_username_password.md]: 1. Enter your SendGrid username.

We will send you an email with account reactivation steps - follow these to access your account again. If you don't receive the email, you may not be entering the correct username.

File: source/_redirects.htaccess [hunk 9]
```diff
@@ -835,4 +835,24 @@ Redirect 301 /Classroom/Troubleshooting/Delivery_Issues/testing_your_connectivit
 Redirect 301 /Classroom/Basics/Email_Infrastructure/smtp_ports.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
 Redirect 301 /Classroom/Basics/Email_Infrastructure/what_is_smtp.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
 Redirect 301 /Classroom/Basics/Email_Infrastructure/recommended_smtp_settings.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
-Redirect 301 /Classroom/Send/How_Emails_Are_Sent/smtp_overview.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
\ No newline at end of file
+Redirect 301 /Classroom/Send/How_Emails_Are_Sent/smtp_overview.html https://sendgrid.com/docs/API_Reference/SMTP_API/getting_started_smtp.html
+
```
Comment (ksigler7) [inline source/_redirects.htaccess]: you added all of the redirects for subusers, but I don't see anything gone for the username/password stuff. I would check the IA, and also do a site search to make sure we didn't miss anything in the IA.

**Comment by ksigler7** [comment]
I'm seeing the account management stuff in two places now. When you moved it to settings, did you delete the old account management folder?

**Comment by ksigler7** [comment]
Also, don't forget to set up weights in the Settings section to get a better page order

File: source/User_Guide/Account_Management/reset_username_password.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+{% endanchor %}
+
+There are two ways to change your password. Through the SendGrid UI and through the Forgot Password page. For more information on password requirements and security see the [password documentation](https://sendgrid.com/docs/Classroom/Basics/Security/password.html). 
+
+{% warning %}
+When you change your account password, your SMTP and API access will change as well. Make sure you update your account information on all systems and software.
+{% endwarning %}
+
+{% anchor h3 %}
+Changing your password through the SendGrid UI
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/reset_username_password.md]: Changing your password

File: source/User_Guide/Account_Management/reset_username_password.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+{% anchor h3 %}
+Changing your password via Forgot Password
+{% endanchor %}
+
+If for some reason you cannot remember your password and cannot log in, you can still change your password. 
+
+*To reset your password using the Forgot Password link:*
+
+1. Navigate to the [SendGrid login page]({{site.site_url}}/login). 
+1. click **[Forgot Password]({{site.site_url}}/user/forgotPassword)**.
```
Comment (ksigler7) [inline source/User_Guide/Account_Management/reset_username_password.md]: Click

File: source/User_Guide/Settings/create_account.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (ksigler7) [inline source/User_Guide/Settings/create_account.md]: boom

File: source/User_Guide/Settings/subusers.md [hunk 13]
```diff
@@ -0,0 +1,199 @@
...
+{% endanchor %}
+
+New SendGrid accounts do not come with Subusers and must be created by the parent account.
+
+*To create a Subuser:*
+
+1. Navigate to **Settings** and select **Subuser Management**. 
+1. Click **Create New Subuser**. 
+1. Enter the following required information into the corresponding fields:
+  * Username - The username your Subuser will use to send email and access the API.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: emails

File: source/User_Guide/Settings/subusers.md [hunk 14]
```diff
@@ -0,0 +1,199 @@
...
+  * Email Address - The contact email address for this Subuser. This must be an active email account as SendGrid may contact a Subuser to provide support,  resolve sending and deliverability issues, or to enforce SendGrid's Terms of Service. Whenever we contact a Subuser, it is our policy to contact the    parent account email address at the same time.
+  * Password - The password your Subuser will use to access SendGrid.com.
+  * Avatar - A [gravatar](http://gravatar.com) will load when an email address is entered, making it easier to identify your Subusers in the Subuser list.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: I think this needs more explanation, it seems really random. "will load" is also passive, if you keeps this, just make it "loads when an..."

File: source/User_Guide/Settings/subusers.md [hunk 15]
```diff
@@ -0,0 +1,199 @@
...
+
+1. Select the IP Address you wish to associate with the Subuser account.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: Assign an IP Address to the Subuser account.

File: source/User_Guide/Settings/subusers.md [hunk 16]
```diff
@@ -0,0 +1,199 @@
...
+  * Email Address - The contact email address for this Subuser. This must be an active email account as SendGrid may contact a Subuser to provide support,  resolve sending and deliverability issues, or to enforce SendGrid's Terms of Service. Whenever we contact a Subuser, it is our policy to contact the    parent account email address at the same time.
+  * Password - The password your Subuser will use to access SendGrid.com.
+  * Avatar - A [gravatar](http://gravatar.com) will load when an email address is entered, making it easier to identify your Subusers in the Subuser list.
+
+1. Select the IP Address you wish to associate with the Subuser account. 
+   When you create a SendGrid account and select a Pro 100k plan or higher, your account is assigned to a dedicated IP address. All of your mail sent through SendGrid will show as originating from this IP address, unless you have purchased additional IPs and assigned IP addresses to them or set up [IP Pools]({{root_url}}/API_Reference/Web_API_v3/IP_Management/ip_pools.html). You can see which other users are assigned to the same IP address, so you can keep similar senders together.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: You are assigned a dedicated IP address when you create a Pro 100k plan or higher. If you send at a high volume, and you're using Subusers to separate out your sending traffic, it's best practice to purchase a separate IP for each sending type.

File: source/User_Guide/Settings/subusers.md [hunk 17]
```diff
@@ -0,0 +1,199 @@
...
+   
+If you need more IP addresses, please contact [SendGrid support]({{site.support_url}}).
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: ~If you need more IP addresses, please contact [SendGrid support]({{site.support_url}}).~ 

I think this probably predates self-service IPs and isn't needed anymore.

File: source/User_Guide/Settings/subusers.md [hunk 18]
```diff
@@ -0,0 +1,199 @@
...
+**Company Phone Number** - The phone number for your Subuser’s business.
+
+**Company Website** - The website for your Subuser’s business.
+
+{% anchor h2 %}
+Allocating or removing credits from a Subuser
+{% endanchor %}
+
+{% warning %}
+This feature only makes changes to a Subuser when the nonrecurring credit option is selected for this Subuser.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: You can only allocate credits by selecting the nonrecurring credit option.

File: source/User_Guide/Settings/subusers.md [hunk 19]
```diff
@@ -0,0 +1,199 @@
...
+{% endwarning %}
+
+If your Subuser has been allocated nonrecurring credits, then this feature will allow you to allocate credits to, or remove credits from this Subuser account. Simply click **Change Credit Rules** and then choose whether to add or subtract credits for this Subuser.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: To allocate Subuser credits, click **Change Credit Rules** and then choose whether to add or subtract credits.

What page is this on? Billing in the parent account?

File: source/User_Guide/Settings/subusers.md [hunk 20]
```diff
@@ -0,0 +1,199 @@
...
+
+{% anchor h2 %}
+Adding a whitelabel domain to a Subuser account
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: Adding a Domain Whitelabel

File: source/User_Guide/Settings/subusers.md [hunk 21]
```diff
@@ -0,0 +1,199 @@
...
+{% endanchor %}
+
+Using a domain whitelabel allows you to better authenticate the email that you send by removing the “sent via sendgrid.net” message that your recipients see on un-whitelabeled emails. This has the added benefit of giving you complete control over your sending reputation.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: Using a domain whitelabel removes the “sent via sendgrid.net” message that your recipients see by authenticating your domain. It also gives you complete control over your sending reputation.

File: source/User_Guide/Settings/subusers.md [hunk 22]
```diff
@@ -0,0 +1,199 @@
...
+
+{% info %}
+In order to add a domain whitelabel or whitelabel link to a Subuser account, there must be one or more of these whitelabels set up in the parent account.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: ~In order~ To

File: source/User_Guide/Settings/subusers.md [hunk 23]
```diff
@@ -0,0 +1,199 @@
...
+{% info %}
+In order to add a domain whitelabel or whitelabel link to a Subuser account, there must be one or more of these whitelabels set up in the parent account. 
+{% endinfo %}
+
+*To add a domain whitelabel to a Subuser account:*
+
+1. Navigate to **Settings** and select **Subuser Management**. 
+1. Locate the Subuser account you wish to delete and click on the name to view the account details. 
+1. On the *Account Settings* page, navigate to the Whitelabel Domain section and then click **Change Sending**. 
+1. From the drop-down menus that appear, select a whitelabel domain and/or a whitelabel link to associate with the Subuser account.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: ~and/or~ and

File: source/User_Guide/Settings/subusers.md [hunk 24]
```diff
@@ -0,0 +1,199 @@
...
+1. Locate the Subuser account you wish to delete and click on the name to view the account details. 
+1. On the *Account Settings* page, navigate to the Whitelabel Domain section and then click **Change Sending**. 
+1. From the drop-down menus that appear, select a whitelabel domain and/or a whitelabel link to associate with the Subuser account. 
+1. Once you've made all of your selections, click **Save**. 
+
+{% anchor h2 %}
+Monitoring Subuser email content
+{% endanchor %}
+
+This feature allows you to recieve a sample of a Subuser's email content to help ensure that the content is not damaging your reputation.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: receive

File: source/User_Guide/Settings/subusers.md [hunk 25]
```diff
@@ -0,0 +1,199 @@
...
+1. From the drop-down menus that appear, select a whitelabel domain and/or a whitelabel link to associate with the Subuser account. 
+1. Once you've made all of your selections, click **Save**. 
+
+{% anchor h2 %}
+Monitoring Subuser email content
+{% endanchor %}
+
+This feature allows you to recieve a sample of a Subuser's email content to help ensure that the content is not damaging your reputation. 
+{% warning %}
+One email credit will be charged for every monitor email that is sent
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: Passive and missing a period.

Each monitor email costs one email credit.

File: source/User_Guide/Settings/subusers.md [hunk 26]
```diff
@@ -0,0 +1,199 @@
...
+To make this easier, we’ve made it so you can impersonate a Subuser from your parent account and then manage the Subuser the same way as if you had logged in with that user’s credentials, without having to log out of your parent account.
+
+*To impersonate a Subuser:*
+
+1. Log into SendGrid. 
+2. Go to the top left corner of the portal screen and click on your name just above the navigation menu.
+3. Click **Switch User**. 
+4. Choose the Subuser you want to impersonate by clicking that Subuser’s name.
+
+The page will refresh and you will notice a message at the top of the screen that says, “You're currently logged in as [Subuser name]”.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: refresh, and

File: source/User_Guide/Settings/subusers.md [hunk 27]
```diff
@@ -0,0 +1,199 @@
...
+
+If you have more than one Subuser you can also switch to another Subuser by clicking **Switch Subuser** in the top-right corner once you are impersonating any Subuser.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: than one Subuser,

File: source/User_Guide/Settings/subusers.md [hunk 28]
```diff
@@ -0,0 +1,199 @@
...
+Export Subuser Information
+{% endanchor %}
+
+If you would like to see how your Subusers are performing in comparison to each other, you can export Subuser data. 
+
+*To export Subuser information:*
+
+1. Navigate to **Settings** and select **Subuser Management**. 
+1. Click **Export Data**. 
+ <br>This downloads a CSV file that will have each Subuser’s reputation, number of requests each Subuser made this month, and the number of requests each Subuser made last month.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: This downloads a CSV file with each Subuser’s reputation, the number of requests they made this month, and the number of requests they made last month.

File: source/User_Guide/Settings/subusers.md [hunk 29]
```diff
@@ -0,0 +1,199 @@
...
+
+ {% anchor h2 %}
+Disabling a Subuser account 
+{% endanchor %}
+
+*To turn off a Subuser’s access to SendGrid temporarily:*
+
+1. Navigate to **Settings** and select **Subuser Management**.  
+1. Locate the Subuser account you wish to disable and click on the name to view the account details. 
+click **Enable**.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: Click

File: source/User_Guide/Settings/subusers.md [hunk 30]
```diff
@@ -0,0 +1,199 @@
...
+
+To re-enable a Subuser’s account, simply click **Disabled**.
```
Comment (ksigler7) [inline source/User_Guide/Settings/subusers.md]: ~simply~

Comment (ksigler7) [reply source/User_Guide/Settings/subusers.md]: Is Disabled correct? My gut says that it's Disable.

File: source/User_Guide/Settings/two_factor_authentication.md [hunk 31]
```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
 You may choose from two different verification methods when using two-factor authentication:
```
Comment (ksigler7) [inline source/User_Guide/Settings/two_factor_authentication.md]: the beginning of this page has a the stutter

SendGrid’s Two-Factor Authentication can be configured to accept a code sent to your mobile phone either via the the Authy App, or an SMS message

File: source/User_Guide/Settings/two_factor_authentication.md [hunk 32]
```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
...
 
 1. Generating a verification code in the [Authy App](https://www.authy.com/app/mobile/).
```
Comment (ksigler7) [inline source/User_Guide/Settings/two_factor_authentication.md]: This is a fragment:

Generate a verification code with the [Authy App](https://www.authy.com/app/mobile/).

File: source/User_Guide/Settings/two_factor_authentication.md [hunk 33]
```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
...
-2. Entering a code sent to your mobile phone that is generated by SendGrid.
+   The Authy App allows you to authenticate over WiFi if you do not have cellular service. However, you may still resort to an SMS message as a       backup if you decide to use Authy.
+1. Entering a code sent to your mobile phone that is generated by SendGrid.
```
Comment (ksigler7) [inline source/User_Guide/Settings/two_factor_authentication.md]: same fragment

File: source/User_Guide/Settings/two_factor_authentication.md [hunk 34]
```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
...
-        The Authy App allows you to authenticate over WiFi if you do not have cellular service. However, you may still resort to an SMS message as a backup if you decide to use Authy.
-      </p>
-    </div>
-    <div class="col-md-6">
-      <img src="{{root_url}}/images/2fa_1.png" class="img-responsive pull-right"/>
-    </div>
-  </div>
-</p>
+{% warning %}
+When authenticating via SMS message, you must have cellular service or you will not be able to log into your account.
```
Comment (ksigler7) [inline source/User_Guide/Settings/two_factor_authentication.md]: service, or

File: source/User_Guide/Settings/two_factor_authentication.md [hunk 35]
```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
...
-
-Once you have installed the Authy App, you will be given a **7-digit token**. Enter this token and click save.
-
-![]({{root_url}}/images/2fa_5.png)
-
-After you have installed the Authy App and entered the 7-digit token, your new Two-Factor Authentication setting is verified and ready for use.
-
-![]({{root_url}}/images/2fa_6.png)
+{% info %}
+You can create multiple configurations of Two-Factor Authentication, allowing you to use more than one mobile device to authenticate when logging into your account
```
Comment (ksigler7) [inline source/User_Guide/Settings/two_factor_authentication.md]: missing period

File: source/User_Guide/Settings/two_factor_authentication.md [hunk 36]
```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
...
+{% endinfo %}
 
-You may back out of the setup process at any time without fear of locking yourself out of your account. We will create an entry labeled “Not Verified”, allowing you to delete the setting without consequence, or verify it for future use.
+*To set up Two-Factor Authentication:*
 
-![]({{root_url}}/images/2fa_7.png)
+1. Navigate to **Settings**, and click **Two-Factor Authentication**. 
+ <br>From this page, you will be able to see an overview of your Two-Factor Authentication settings, along with any settings that you have created for Credentialed Users.
```
Comment (ksigler7) [inline source/User_Guide/Settings/two_factor_authentication.md]: ~will be able to~

File: source/User_Guide/Settings/two_factor_authentication.md [hunk 37]
```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
...
+{% info %}
+Subusers and Teammates need to create their own Two-Factor Authentication settings in their account using the same steps. 
+{% endinfo %}
 
 {% anchor h2 %}
 Using Two-Factor Authentication
 {% endanchor h2 %}
 
-After you have created a setting for Two-Factor Authentication, simply log in as you normally would. If you have selected to receive a text message with your authentication token, look for the text message and enter the code that you receive. If you have selected the Authy App, go to your app, and enter the generated code.
+After you have created a setting for Two-Factor Authentication, log in as you normally would. If you have selected to receive a text message with your authentication token, look for the text message and enter the code that you receive. If you have selected the Authy App, go to your app, and enter the generated code.
```
Comment (ksigler7) [inline source/User_Guide/Settings/two_factor_authentication.md]: ~have~ ~have~

File: source/User_Guide/Settings/two_factor_authentication.md [hunk 38]
```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
...
 
-Once setup, you will always be required to use Two-Factor Authentication to perform security-restricted actions in your SendGrid account, such as logging in or changing your Two-Factor Authentication settings. When prompted, simply enter the 7-digit token sent to your device, or generated by the Authy App.
+Once set up, you will always be required to use Two-Factor Authentication to perform security-restricted actions in your SendGrid account, such as logging in or changing your Two-Factor Authentication settings. When prompted, simply enter the 7-digit token sent to your device, or generated by the Authy App.
```
Comment (ksigler7) [inline source/User_Guide/Settings/two_factor_authentication.md]: ~simply~

File: source/_redirects.htaccess (hunk 9 above)
Comment (ksigler7) [reply source/_redirects.htaccess]: + 1 any for 2FA?

**Comment by thinkingserious** [comment]
Hello @Whatthefoxsays,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**